JavaScript/README.md

```diff
@@ -17,3 +17,4 @@
   2. [Command-Line](Topics/4.Command_Line)
 - [Syntax](Topics/5.Syntax)
 - [Hoisting](Topics/6.Hoisting)
+- [Promises](Topics/7.Promises)
```

Comment:
Please avoid this change, because I'll be editing the index manually and this may lead to conflicts.

Reply:
Actually I did not had any idea about this I am really sorry
Okay will do the changes and will make youtube link also
@paramsiddharth